root/report/CDTocNotApplied.js

```diff
@@ -23,34 +20,26 @@ const CDTocNotApplied = ({
   generated,
   items,
   pager,
-}: ReportDataT<ReportCDTocReleaseT>): React.Element<typeof Layout> => (
-  <Layout $c={$c} fullWidth title={l('Disc IDs attached but not applied')}>
-    <h1>{l('Disc IDs attached but not applied')}</h1>
-
-    <ul>
-      <li>
-        {l(`This report shows disc IDs attached to a release but obviously not
-        applied because at last one track duration is unknown on the release.
-        The report is also restricted to mediums where only one disc ID is
-        attached, so it is highly likely that the disc ID can be applied
-        without any worries. Do make sure though that no existing durations
-        clash with the disc ID, or that any clashes are clear mistakes.`)}
-      </li>
-      <li>
-        {texp.l('Total discIDs found: {count}',
-                {count: pager.total_entries})}
-      </li>
-      <li>
-        {texp.l('Generated on {date}',
-                {date: formatUserDate($c, generated)})}
-      </li>
-
-      {canBeFiltered ? <FilterLink $c={$c} filtered={filtered} /> : null}
-    </ul>
-
+}: ReportDataT<ReportCDTocReleaseT>): React.Element<typeof ReportLayout> => (
+  <ReportLayout
+    $c={$c}
+    canBeFiltered={canBeFiltered}
+    description={l(
+      `This report shows disc IDs attached to a release but obviously not
+       applied because at last one track duration is unknown on the release.
+       The report is also restricted to mediums where only one disc ID is
+       attached, so it is highly likely that the disc ID can be applied
+       without any worries. Do make sure though that no existing durations
+       clash with the disc ID, or that any clashes are clear mistakes.`,
+    )}
+    entityType="release"
+    filtered={filtered}
+    generated={generated}
+    title={l('Disc IDs attached but not applied')}
+    totalEntries={pager.total_entries}
+  >
```

Comment:
`'Total discIDs found: {count}'` is being replaced with `'Total releases found: {count}'`.

It is a bit unclear whether or not releases with several media can be reported several times.

Reply:
This is fair, and in fact the other discID report (which currently says "releases found") also should be changed to say discIDs. Fixed.